src/components/Search/SelectedRevisions.tsx

```diff
@@ -28,47 +27,19 @@ function SelectedRevisions({
   isWarning,
   formIsDisplayed,
   isEditable,
+  revisions,
+  repositories,
 }: SelectedRevisionsProps) {
   const styles = SelectRevsStyles(mode);
   const location = useLocation();
   const view = location.pathname == '/' ? searchView : compareView;
-  const [revisions, setRevisions] = useState<RevisionsList[]>([]);
-  const [repositories, setRepositories] = useState<Repository['name'][]>([]);
-  const checkedRevisionsList = useAppSelector(
-    (state) => state.search[searchType].checkedRevisions,
-  );
-
-  const checkedRepositories = checkedRevisionsList.map((item) => {
-    const selectedRep = repoMap[item.repository_id];
-    return selectedRep;
-  });
-  const selectedRevisions = useAppSelector(
-    (state) => state.selectedRevisions.revisions,
-  );
-  const compareViewRepositories = selectedRevisions.map((item) => {
-    const selectedRep = repoMap[item.repository_id];
-    return selectedRep;
-  });
-  const compareViewSelectedRevisions = useAppSelector(
-    (state) => state.selectedRevisions[searchType],
-  );
-
-  useEffect(() => {
-    if (view === searchView) {
-      setRevisions(checkedRevisionsList);
-      setRepositories(checkedRepositories as Repository['name'][]);
-    } else {
-      setRevisions(compareViewSelectedRevisions);
-      setRepositories(compareViewRepositories as Repository['name'][]);
-    }
-  }, [checkedRevisionsList, selectedRevisions]);
 
   return (
     <Box
       className={`${styles.box} ${searchType}-box`}
       data-testid={`selected-revs-${view}`}
     >
-      <List>
+      <List key={null}>
```

Comment:
why this key?

Reply:
Oh yeah that was some nonsense I was play around with that I forgot to remove. Thanks for catching.